webapp/channels/src/components/widgets/modals/components/save_changes_panel.tsx

```diff
@@ -81,33 +99,34 @@ function SaveChangesPanel({handleSubmit, handleCancel, handleClose, tabChangeErr
             );
         }
 
+        const saveButtonDisabled = tabChangeError || state === 'error';
+
         return (
-            <div className='mm-save-changes-panel__btn-ctr'>
+            <div className='SaveChangesPanel__btn-ctr'>
                 <button
-                    data-testid='mm-save-changes-panel__cancel-btn'
+                    data-testid='SaveChangesPanel__cancel-btn'
                     className={cancelButtonClassName}
                     onClick={handleCancel}
                 >
-                    <FormattedMessage
-                        id='saveChangesPanel.cancel'
-                        defaultMessage='Undo'
-                    />
+                    {cancelButtonText || (
+                        <FormattedMessage
+                            id='saveChangesPanel.cancel'
+                            defaultMessage='Undo'
+                        />
+                    )}
                 </button>
                 <button
-                    data-testid='mm-save-changes-panel__save-btn'
+                    data-testid='SaveChangesPanel__save-btn'
                     className={saveButtonClassName}
                     onClick={handleSubmit}
+                    disabled={saveButtonDisabled}
                 >
-                    {state === 'error' ?
-                        <FormattedMessage
-                            id='saveChangesPanel.tryAgain'
-                            defaultMessage='Try again'
```

Comment:
Why remove this case?

Reply:
the new design on error state shows the Save button disabled with 0.5 opacity.